packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -72,14 +71,26 @@ const getSchemaFromEditorForm = (
   };
 };
 
-// @todo loading state
 const Page: NextPageWithLayout = () => {
   const router = useRouter();
+
   // @todo how to handle remote types
   const isDraft = !!router.query.draft;
-  const baseEntityTypeUri = isDraft ? null : getBaseUri(router.asPath);
   const namespace = useRouteNamespace();
 
+  const entityTypeId = router.query["entity-type-id"];
+  const baseEntityTypeUri =
+    !isDraft && namespace?.shortname
+      ? getBaseUri(
+          entityTypeId && !Array.isArray(entityTypeId) ? entityTypeId : "",
+          namespace.shortname,
+        )
+      : null;
+
+  const entityTypeEntitiesValue = useEntityTypeEntitiesContextValue(
```

Comment:
The backend team still doesn't seem to have decided on how to tackle this ([slack thread](https://hashintel.slack.com/archives/C03F7V6DU9M/p1668775656636149?thread_ts=1668775226.165989&cid=C03F7V6DU9M)).
However, I think we can create a resolver to just return the count when we can actually query only the entities belonging to an entity type, instead of getting them all and filtering them in the client. Here's the [task](https://app.asana.com/0/0/1203400386993345/f) I created to track this.